cli/bin/build-go.sh

```diff
@@ -0,0 +1,58 @@
+#!/bin/bash
+
+set +x
+
+BIN_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+cd $BIN_DIR/..
```

Comment:
It *should* never happen, but `$BIN_DIR` could contain a space, so let's quote it. I think it's worth being maximally defensive in Bash scripts.